config/routes.rb

```diff
@@ -186,9 +186,44 @@
 
   # lectures routes
 
-  get "lectures/:id/food",
+  get "lectures/:id/material",
       to: "media#index",
-      as: "lecture_food"
+      as: "lecture_material"
+
+  # New semantic routes replace the old food routes
+  get "lectures/:id/lesson_materials",
+      to: "media#index",
+      as: "lecture_lesson_materials",
+      defaults: { project: "lesson_material" }
+
+  get "lectures/:id/script",
+      to: "media#index",
+      as: "lecture_script",
+      defaults: { project: "script" }
+
+  get "lectures/:id/exercises",
+      to: "media#index",
+      as: "lecture_exercises",
+      defaults: { project: "exercise" }
+
+  get "lectures/:id/quizzes",
+      to: "media#index",
+      as: "lecture_quizzes",
+      defaults: { project: "quiz" }
+
+  get "lectures/:id/worked_examples",
+      to: "media#index",
+      as: "lecture_worked_examples",
+      defaults: { project: "worked_example" }
+
+  get "lectures/:id/repetitions",
+      to: "media#index",
+      as: "lecture_repetitions",
+      defaults: { project: "repetition" }
+
+  get "lectures/:id/miscellaneous",
+      to: "media#index", as: "lecture_supplements",
```

Comment:
Why not `lecture_miscellaneous` here? Furthermore break `as:` to a new line like for the other entries here.

Reply:
Indeed, `lecture_miscellaneous` makes more sense. Fixed.